microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
 
-![Click the highlighted toggle button to expand the Metrics items](../media/speed-glitch-tony-remix-audits-performance-metrics-expanded.msft.png)
+![An expanded opportunity, with more information, and a learn more link](./images/expanded-opportunity.png)
 
-Below Metrics is a collection of screenshots that show you how the page looked as it loaded.
+###### Diagnostics
 
-![How the page looked while loading](../media/speed-glitch-tony-remix-audits-performance-view-trace.msft.png)
+The **Diagnostics** section provides more information about factors that contribute to the load time of the page:
 
-The **Opportunities** section provides specific tips on how to improve the load performance of this specific page.
+![The Diagnostics section](./images/diagnostics-section.png)
 
-![The Opportunities section](../media/speed-glitch-tony-remix-audits-performance-view-trace.msft.png)
+###### Passed audits
 
-Click an opportunity to learn more about it.
+The **Passed audits** section shows you what the site is doing correctly.  Click **Show** to expand the section:
 
-![Eliminate render-blocking resources opportunity](../media/speed-glitch-tony-remix-audits-performance-opportunities-expanded.msft.png)
+![The Passed Audits section](./images/passed-audits-section.png)
 
-Click **Learn More** to display documentation about why an opportunity is important, and specific recommendations on how to fix it.
 
-![Documentation for the Eliminate render-blocking resources opportunity](../media/speed-web-dev-performance-audits.msft.png)
+<!-- ====================================================================== -->
+## Step 3: Experiment
 
-The **Diagnostics** section provides more information about factors that contribute to the load time of the page.
+The **Opportunities** section of your report gives you tips on how to improve the performance of the webpage.  In this section, you implement the recommended changes to the codebase, auditing the site again after each change to measure how it affects site speed.
 
-![The Diagnostics section](../media/speed-glitch-tony-remix-audits-performance-diagnostics.msft.png)
+#### Resize images
 
-The **Passed Audits** section shows you what the site is doing correctly.  Click to expand the section.
+Your report indicates that serving appropriately sized images is one of the top opportunities for improving the performance of the page.  Resizing images helps reduce the size of the network payload.  If your user is viewing your images on a mobile device screen that is 500-pixels-wide, there's really no point in sending a 1500-pixel-wide image.  Ideally, you send a 500-pixel-wide image, at most.
 
-![The Passed Audits section](../media/speed-glitch-tony-remix-audits-performance-passed-audits.msft.png)
+In the **Opportunities** section of your report, click **Properly size images** to display which images to resize. **Lighthouse** lists four `.jpg` files you should resize to improve load time:
```

Comment:
```suggestion
In the **Opportunities** section of your report, click **Properly size images** to display which images to resize. **Lighthouse** lists four `.jpg` files that you can resize to improve load time:
```
possibly keep more optional gentle wording, IF it's conceivable that Dev might choose better resolution by keeping existing size.  tech review this change.  Is it always "should", or depends on crispness of the particular file?